src/app/app-modules/core/services/session-storage.service.ts

```
@@ -0,0 +1,21 @@
+  userName:any;
+  username:any;
+  fullName:any;
+  facilityID:any;
+  facilityDetail:any;
+  benFlowID:any;
+  healthID:any;
+  providerServiceID:any;
+  services:any;
+
+  
+  constructor() { }
```

Biome

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)